README.md

```diff
@@ -9,10 +9,18 @@ The purpose of *ddev* is to support developers with a local copy of a site for d
 You can see all "ddev" usages using the help commands, like `ddev -h`, `ddev start -h`, etc.
 
 ## System Requirements
-- MacOS Sierra or Linux*
-- [docker](https://www.docker.com/community-edition)
 
-* *Currently only tested on Ubuntu, please [let us know](https://github.com/drud/ddev/issues/new) if you hit an issue on another flavor.*
+- [docker](https://www.docker.com/community-edition)
+- OS Support
+  - macOS Sierra (fully supported with automated tests)
+  - Linux
+    * Ubuntu 16.04 LTS (fully supported with automated tests)
+    * Debian Jessie (tested manually with automated tests planned)
+    * Fedora 25 (tested manually with automated tests planned)
+  - Windows 10 Pro (**experimental support!**)
```

Comment:
If we add this, do we need to specifically update the language for our install script to indicate OSX/Linux only?